src/ConventionalCommitUtilties.js

```diff
@@ -0,0 +1,61 @@
+import ChildProcessUtilities from "./ChildProcessUtilities";
+import FileSystemUtilities from "./FileSystemUtilities";
+import logger from "./logger";
+import semver from "semver";
+
+const CHANGELOG_HEADER = `# Change Log
+
+All notable changes to this project will be documented in this file.
+See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.`;
+
+export default class ConventionalCommitUtilties {
+  @logger.logifySync()
+  static recommendVersion(name, version) {
+    try {
+      const pkgPath = `./packages/${name}/`;
+      const recommendedBump = ChildProcessUtilities.execSync(
+        `${require.resolve("conventional-recommended-bump/cli.js")} -l ${name} --commit-path=${pkgPath} -p angular`
+      );
+      return semver.inc(version, recommendedBump);
+    } catch (err) {
+      return semver.inc(version, "patch");
+    }
+  }
+
+  @logger.logifySync()
+  static updateChangelog(name, changelogLocation) {
+    try {
+      const pkgPath = `./packages/${name}/`;
+      let changelogContents = "";
+      if (FileSystemUtilities.existsSync(changelogLocation)) {
+        changelogContents = FileSystemUtilities.readFileSync(changelogLocation);
+      }
+
+      // run conventional-changelog-cli to generate the markdown
+      // for the upcoming release.
+      const newEntry = ChildProcessUtilities.execSync(
+        `${require.resolve("conventional-changelog-cli/cli.js")} -l ${name} --commit-path=${pkgPath} --pkg=${pkgPath} -p angular`
```

Comment:
It was my feeling that relying on the `conventional-recommended-bump` cli, and `conventional-changelog-cli` fit well with the existing patterns in the codebase; simply shelling out to `npm` with `NpmUtilities.js` , or git with `GitUtilities.js`:

* this minimized the number of modules from the conventional CHANGELOG ecosystem that I would need to pull in to support this functionality.
* and allows me, while adding tests, to use the same helpers/conventions used when adding tests to `npm`/`git`:
  * it's my plan to mock `ChildProcess`, using the same approach as here: https://github.com/lerna/lerna/blob/master/test/PublishCommand.js#L1707